### DISCUSSION TITLE
How to spawn heterogenoeus environments

### DISCUSSION THREAD

**Discussion by lopatovsky** [isaac-sim/IsaacLab#952] [Q&A] [answered]
Imagine a usecase similar to franka_cabinet, but we have a usd file for 16 different cabinets. 
Now we want to run a training with 2048 environments, where each cabinet would be spawned in (2048/16 = 128) envs. 

What is the recommended way to tackle such a usecase?

**Comment by Mayankm96** [comment, marked as answer]
Please check #186
